### DISCUSSION TITLE
Weight and BMI are derived parameters?

### DISCUSSION THREAD

**Discussion by moPellowe** [Open-Systems-Pharmacology/Forum#1144] [Questions & Problems] [answered]
How can both weight and BMI both be derived parameters? I thought that you would've needed at least two of the three (height, weight, BMI) to calculate the remaining one. What am I missing here?

**Comment by msevestre** [comment, marked as answer]
What are you referring to? What do you mean with derived parameters?

**Reply by moPellowe** [reply]
I've created individuals using createIndividual() in the ospsuite-R package with age, gender, weight, and height as input. When I look at the individuals afterwards, I noticed that age and height were distributed parameters, weight was a derived parameter, and gender was not saved. I'm just wondering why weight is a derived parameter.

**Reply by msevestre** [reply]
I see. Context is everything...

Weight is a formula and is the sum of all organ volumes. It should not be used to update an individual otherwise you'll overwrite a formula with a constant.
It is outputted as derived parameters however so that you can use it for other calculations. But strictly speaking, this parameter value is not required to update a simulation. Does it make sense?

**Reply by moPellowe** [reply]
Okay, so why or how is it used as an input in the createIndividualCharacteristics function?

**Reply by msevestre** [reply]
This function is only a wrapper to what PK-Sim does. When you create an individual, you need to specify the bodyweight
https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-creating-individuals#biometrics

Otherwise, PK-Sim cannot create an individual. The fact that the parameter is a derived parameters also means that it was made available for users to use in a convenient way (e..g you do not have to memorize it somewhere else). If all goes well, InputBW (what you give to the createIndividual method) == DerivedBW (formula as Sum of all organ weights)

**Reply by moPellowe** [reply]
This makes sense to me. Thank you for explaining, @msevestre . I appreciate your help.
